compatibility/WORKSPACE

```diff
@@ -163,7 +163,7 @@ dev_env_tool(
 
 nixpkgs_package(
     name = "postgresql_nix",
-    attribute_path = "postgresql_9_6",
+    attribute_path = "postgresql_12",  # requirement of the partitioned schema
```

Comment:
It's probably worth mentioning in the documentation. Would this represent a problem for others working with SQL-backed ledgers?

Reply:
For the final rollout we might stick to manual partitioning. (context: this bump of the postgresql version is needed because current schema using postgre-specific declarative partitioning) This might would mean no version increase eventually.
Otherwise very good comment: we have now uniform ephemeral postgreSQL support, if we want to differentiate, that would require some extra work.